solidity/utils/test_deployment.js

```diff
@@ -290,11 +293,14 @@ const run = async () => {
                 weights
             )
         );
+        console.log('BBB');
         const converterAnchor = deployed(
             web3,
             'IConverterAnchor',
             await converterRegistry.methods.getAnchor(index).call()
         );
+        console.log('CCC');
+
```

Comment:
You forgot to remove your debug leftovers.

Reply:
Oops